ee-feature-pack/ee-10-api/src/main/resources/modules/system/layers/base/jakarta/security/enterprise/api/main/module.xml

```diff
@@ -41,5 +41,11 @@
         <module name="javax.annotation.api" />
         <module name="javax.inject.api" />
         <module name="javax.json.api" />
+        <!--TODO move the WELD dependencies to implementation module instead, see comments on https://github.com/wildfly/wildfly/pull/16155 for more details
```

Comment:
Nitpicker's note - impl module already has these deps, so once it is solved, these can be simply erased.

Reply:
@manovotn I think the org.jboss.weld.api is missing there, but I updated the message to be more clear. Thanks!